content/events/23-24/verkiezingen.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Zeus zoekt voorzitter + Open vergadering
+description: Wie wordt het volgende glorizeuse voorzitter?
+created_at: 28-09-2023
+time: '05-10-2023 18:00'
+end: '05-10-2023 22:00'
+location: 'De Kelder'
+locationlink: $kelder
+banner: https://pics.zeus.gent/6x3s9HZzUft9M8ofOI2MHJvh016eeLq6fheKrWcB.jpg
```

Comment:
Misschien een andere  banner kiezen, want het event gaat niet door in de studententuin.